docs/sharded-computation.ipynb

```diff
@@ -13,21 +13,44 @@
     "\n",
     "The tutorial covers three modes of parallel computation:\n",
     "\n",
-    "- _Automatic parallelism via {func}`jax.jit`_: The compiler chooses the optimal computation strategy (a.k.a. \"the compiler takes the wheel\").\n",
-    "- _Semi-automated parallelism_ using {func}`jax.jit` and {func}`jax.lax.with_sharding_constraint`\n",
```

Comment:
Why delete this?

Reply:
Cause I made the notebook to execute and that requires having CPU devices. This should be in a different doc anyways. It doesn't fit in here.